### PR TITLE
Fix: Signup through Social Account available on signup page

### PR DESCRIPTION
**Pull request by DESHRAJ**

### CODE REVIEW

**Comment by vnbang2003** [comment]
@DESHRAJ Do you mind to share screenshot and provide a bit description on what all functionality is cover in this pull request.

**Comment by vnbang2003** [comment]
@DESHRAJ Can we just add screenshot as it a UI change also so we can provide feedback if something need to fixed upfront.

**Comment by kirang89** [comment]
Travis CI is reporting a syntax error:
```
./wye/profiles/adapter.py:32:19: E901 SyntaxError: invalid syntax
```

**Comment by vnbang2003** [comment]
@DESHRAJ Can i get fix with clean else i might need to reject this pull request ?

**Comment by DESHRAJ** [comment]
Hey @vnbang2003, Sorry I was not able to work on this bug since I am busy with my college exams. I will fix this bug by this weekend.

File: wye/profiles/adapter.py [hunk 1]
```diff
@@ -0,0 +1,33 @@
...
+            email = sociallogin.account.extra_data['email'].lower()
+            email_address = EmailAddress.objects.get(email__iexact=email)
+
+        # if it does not, let allauth take care of this new social account
+        except EmailAddress.DoesNotExist:
+            return
+
+        # if it does, connect this new social login to the existing user
+        user = email_address.user
+        print user
```
Comment (aniketmaithani) [inline wye/profiles/adapter.py]: I think you should use 
`print (user)`

**Comment by vnbang2003** [comment]
@DESHRAJ Can i expect update pull request today?

**Comment by DESHRAJ** [comment]
Sorry for being late. I have fixed the issue #179 in this pull request. I will create another pull request for the #178 and #180

**Comment by vnbang2003** [comment]
@DESHRAJ Build is failing can you please take a look and fix it.

**Comment by vnbang2003** [comment]
@DESHRAJ Can you please do needful to close this PR it been very long time pending.
I worst case once of us has to rewrite code and fix it.

**Comment by DESHRAJ** [comment]
@vnbang2003 sorry for being late. Now, I have made changes.

**Comment by vnbang2003** [comment]
@DESHRAJ test coverage is down. Can you please add test case for you changes so i can accept pull request and close it.

**Comment by DESHRAJ** [comment]
@vnbang2003 can you help me out with why the coveralls failing ?

**Comment by vnbang2003** [comment]
@DESHRAJ You have added 4 lines of code with no test case to test them hence it reducing test coverage.
You can add test case or change testcase in https://github.com/pythonindia/wye/blob/master/tests/functional/test_user_flow.py.

On you local once you run test with command  "py.test -s --cov-report=html --cov" it should coverage report locally, Look for htmlcov/index.html file 
Once your testcase cover the latest changes it will automatically increase coverage percentage on your local machine.

**Comment by DESHRAJ** [comment]
@vnbang2003 Its just the change in labels of the fields on the signup page. I don't understand that what test case can be written for this. Should I check that if the labels contain * for the required fields in the tests ?

**Comment by vnbang2003** [comment]
@DESHRAJ for now i will accept and close the PR as 0.01% looks fine to me.

**Pull request merged**